tests/sklearn/test_sklearn_autolog.py

```diff
@@ -990,3 +992,43 @@ def test_autolog_does_not_throw_when_infer_signature_fails():
     model_conf = get_model_conf(run.info.artifact_uri)
     assert "signature" not in model_conf.to_dict()
     assert "saved_input_example_info" not in model_conf.to_dict()
+
+
+def test_autolog_configuration_options():
```

Comment:
We should also adopt @harupy 's suggestion from https://github.com/mlflow/mlflow/pull/3449/files#r494000413 within this sklearn tests